core/extensions/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
Shouldn't header look like below?
```
/**
 * Copyright Soramitsu Co., Ltd. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */```

Reply:
In CMakeLists file comments are created via `#` symbol, not `/** */`